docs/reference/ingest/processors/inference.asciidoc

```diff
@@ -0,0 +1,77 @@
+[role="xpack"]
+[testenv="basic"]
+[[inference-processor]]
+=== {infer-cap} Processor
+
+Uses a pre-trained {dfanalytics} model to infer against the data that is being 
+ingested in the pipeline.
+
+
+[[inference-options]]
+.{infer-cap} Options
+[options="header"]
+|======
+| Name               | Required  | Default                        | Description
+| `model_id   `      | yes       | -                              | (String) The ID of the model to load and infer against.
+| `target_field`     | yes       | `ml.inference.<processor_tag>` | (String) The field name to which to write the results objects.
+| `field_mappings`   | no        | -                              | (Object) Maps the document field names to the known field names of the model.
```

Comment:
I think this is required...if testing showed otherwise, I think that should be fixed.

Reply:
The two required field mistakes are copy/paste errors. Thanks!